src/playground/with-layout/toggle/toggle-status.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'nb-toggle-status',
+  template: `
+    <nb-card>
+      <nb-card-body>
+        <div>
```

Comment:
Don't you mind removing extra `div`s from all toggle examples? It'll greatly improve readability.
![image](https://user-images.githubusercontent.com/6151971/61786366-30a2cd00-ae16-11e9-8d05-69478071485a.png)
vs.
![image](https://user-images.githubusercontent.com/6151971/61786381-34ceea80-ae16-11e9-9b38-a6e36ab77e86.png)

Instead of `div`s you could add `example-items-col` class to the card body. It'll line up toggles in column and add necessary margins.